plugins/scanners/scanoss/src/main/kotlin/ScanOssResultParser.kt

```diff
@@ -63,7 +64,13 @@ internal fun generateSummary(startTime: Instant, endTime: Instant, results: List
 
                 MatchType.snippet -> {
                     val localFile = requireNotNull(result.filePath)
-                    snippetFindings += createSnippetFindings(details, localFile)
+                    if (details.status == StatusType.pending) {
+                        snippetFindings += createSnippetFindings(details, localFile)
+                    } else {
+                        logger.warn { "File '$localFile' is identified, not including on snippet findings" }
```

Comment:
Nits:
- "on" -> "in"
- Missing dot at end of sentence.

Reply:
Changed also from `warn` to `info` log level